subprojects/common/logger/src/main/kotlin/com.avito.logger/Logger.kt

```diff
@@ -2,6 +2,8 @@ package com.avito.logger
 
 interface Logger {
     fun debug(msg: String)
+    // TODO: implement in clients and remove empty implementation
+    fun info(msg: String) {} // empty implementation is for backward compatibility
```

Comment:
Had to add a default implementation for backward compatibility in clients. They implement the Logger.